### ISSUE TITLE
timesaver for crafting doesn't seem to respect crafting speed bonuses

### ISSUE THREAD

**Issue by Xetaxheb** [ZwerOxotnik/zk-lib#5]
for easy example rpg system https://mods.factorio.com/mod/RPGsystem has a bonus to crafting speed (character_crafting_speed_modifier) but if timesaver is enabled it is completely ignored

**Comment by ZwerOxotnik** [comment]
Yep, probably, if I'll use the same method as in https://github.com/ZwerOxotnik/zk-lib/blob/1.0/addons/adrenaline.lua it'll be more compatible with other mods.
